base/linalg/uniformscaling.jl

```diff
@@ -371,3 +371,7 @@ UniformScaling{Float64}
 ```
 """
 chol(J::UniformScaling, args...) = ((C, info) = _chol!(J, nothing); @assertposdef C info)
+
+## Diagonal construction from UniformScaling
+Diagonal{T}(s::UniformScaling, m::Integer) where {T} = Diagonal{T}(fill(T(s.λ), m))
+Diagonal(s::UniformScaling, m::Integer) where {T} = Diagonal{eltype(s)}(s, m)
```

Comment:
Missing `{T}` on `UniformScaling`, so either add that and change `eltype(s)` to `T` or simply remove `where {T}`

Reply:
A beautiful example of why one shouldn't make "trivial changes" without retesting 😄. Thanks `@fredrikekre`!